index.js

```diff
@@ -141,7 +133,7 @@ function validateTimestamp(requestBody) {
 
 
 // certificate validator express middleware for amazon echo
-var verifier = module.exports = function(cert_url, signature, requestBody, callback) {
+var verifier = module.exports = function(cert_url, signature, requestBody, callback, cache) {
```

Comment:
Use case: better/easier unit-testability.
Each test can pass in a cache (either clean or already populated, depending on what path is being tested).